### PR TITLE
Create Saturn2357-Portland2

### PR DESCRIPTION
**Pull request by Saturn2357**

### CODE REVIEW

File: _pins/Saturn2357-Portland2 [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (Aric22Bear) [inline _pins/Saturn2357-Portland2]: these lines should be removed, just keep three lines which contain your username, latitude and longitude, and then that will work

Comment (Saturn2357) [inline _pins/Saturn2357-Portland2]: That worked! :-) However my icon isn't showing up on the map after I have merged. Is there something I am doing wrong?

Comment (jr-cologne) [inline _pins/Saturn2357-Portland2]: The lines mustn't be removed. Put three of them to the top and three of them to the bottom.

Then your pin should show up correctly.

Comment (jr-cologne) [inline _pins/Saturn2357-Portland2]: Like that:

```
---
githubHandle: Saturn2357
latitude: 45.466388
longitude:-122.626374
---
```

Comment (Saturn2357) [inline _pins/Saturn2357-Portland2]: Once I put the lines back in place, Travis CI (which I have no clue about) rejects my merge once again. There must be some trick in getting the pin on the map. Did I miss anything on the tutorial?

**Pull request merged**